.github/workflows/galaxy.yml

```
@@ -11,8 +11,28 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
+    # deploy the collection first, because if it fails, we don't want
+    # to update the galaxy.yml
     - name: Deploy the collection
       uses: artis3n/ansible_galaxy_collection@v2
       with:
         api_key: ${{ secrets.GALAXY_API_KEY }}
         galaxy_version: ${{ github.event.release.tag_name }}
+
+    - name: update galaxy.yml with new version
+      uses: microsoft/variable-substitution@v1
+      with:
+        files: 'galaxy.yml'
+      env:
+        version: "${{ github.event.release.tag_name }}"
+
+    - name: push changelog
```

the name should be updated to reflect the file we are dealing with
```suggestion
    - name: push galaxy.yml
```